src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2888,7 +2889,7 @@ void Creature::UpdateMovementFlags()
     // Set the movement flags if the creature is in that mode. (Only fly if actually in air, only swim if in water, etc)
     float ground = GetFloorZ();
 
-    bool isInAir = (G3D::fuzzyGt(GetPositionZMinusOffset(), ground + 0.05f) || G3D::fuzzyLt(GetPositionZMinusOffset(), ground - 0.05f)); // Can be underground too, prevent the falling
+    bool isInAir = (G3D::fuzzyGt(GetPositionZMinusOffset(), ground) || G3D::fuzzyLt(GetPositionZMinusOffset(), ground)); // Can be underground too, prevent the falling
```

Comment:
what was the purpose of +/- 0.05 fuzzyGt/Lt and why is it removed now ?

Reply:
It was definitely intended here, don't want creatures toggling flight/walk like mad based on inaccurancies during spline update

Reply:
Will place it back then, but I would like to have those 0.05f's defined somewhere for consistency, any good place for it ?

Reply:
SharedDefines.h is not ok to store such?